networkx/drawing/nx_pylab.py

```diff
@@ -261,7 +271,7 @@ def draw_networkx(G, pos=None, with_labels=True, **kwds):
         pos = nx.drawing.spring_layout(G)  # default to spring layout
 
     node_collection = draw_networkx_nodes(G, pos, **kwds)
-    edge_collection = draw_networkx_edges(G, pos, **kwds)
+    edge_collection = draw_networkx_edges(G, pos, arrows=arrows, **kwds)
```

Comment:
Are these two variables `node_collection` and `edge_collection` really needed? They are not used, so just calling the drawing functions would be better.